config/config-windows.yaml

```diff
@@ -263,6 +291,63 @@ processors:
           # change data type from int64 -> double
           - action: toggle_scalar_data_type
 
+  # filter to include only agent metrics supported by cloud monitoring
+  filter/agent:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - otelcol_process_uptime
+          - otelcol_process_memory_rss
+          - otelcol_grpc_io_client_completed_rpcs
+          - otelcol_otelcol_googlecloudmonitoring_point_count
```

Comment:
Why do we still have double `otelcol_` prefix in self-observability metrics, and where is it coming from? 
I expected naming like `otelcol_exporter_googlecloudmonitoring_point_count` generated by "new metrics" mechanism like in https://github.com/open-telemetry/opentelemetry-collector/blob/95389af250779aa176a2051ae1c26d7c9fab26cd/obsreport/obsreport_exporter.go#L50.

Reply:
I see, it's defined in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/31c0d95d7cf9e1b2a1601bbdca6889d7e43ddd11/exporter/stackdriverexporter/observability.go#L29

I think we need to fix the naming there to be consistent with the built-in telemetry metrics.

Reply:
Yes its a bug. I didn't realize the prefix was already added and didn't verify before merging that PR in Contrib 😦  I will fix that in Contrib and then update this later.

For this PR I will leave the config as is though so the transform works for now

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/906